src/pages/leaderboard.tsx

```diff
@@ -1,9 +1,34 @@
+import TopUserCard from '@/components/leaderboard/TopThreeCard';
```

Comment:
If you look at the file `src/components/auth/index.ts`, we should be following a similar structure in all of the folders in the project that contain components and other exports so let's make a file `src/components/leaderboard/index.ts` and export components the way we do there. This let's us use named exports from a single location and is a good way to manage all of the pieces in our application.

Once you have the export set up like that, the import here will look like
```
import { TopThreeCard } from '@/components/leaderboard';
```